appliance-root/opt/noit/prod/etc/circonus-modules.conf

```diff
@@ -28,8 +28,7 @@
       <lua_function>start_upkeep</lua_function>
     </config>
   </generic>
-
+  <include file="circonus-modules-general.conf" snippet="true" readonly="true"/>
   <include file="circonus-modules-site.conf" snippet="true" readonly="true"/>
   <include file="circonus-modules-enterprise.conf" snippet="true" readonly="true"/>
-  <include file="circonus-modules-general.conf" snippet="true" readonly="true"/>
```

Comment:
`resmon` needs to be loaded before `consul`.. since consul is in `enterprise` it would not load